root/RawLogsScraper.py

```diff
@@ -225,8 +225,12 @@ def roundAge(round):
     # If you're not a string (Not a round id) I'm not interested
     if round.isnumeric():
         return int(round)
-    
-    id = round.split("-")[1]
```

Comment:
i forgot what exactly made a bad index runtime here but now it cant

Reply:
it'd be photo folders, fixed with a hardcoded round id directory access, but irrespective of that this is honestly a good idea